packages/ghidra.vm/tools/chocolateyuninstall.ps1

```diff
@@ -0,0 +1,7 @@
+$ErrorActionPreference = 'Continue'
+Import-Module vm.common -Force -DisableNameChecking
+
+$toolName = 'ghidra'
+$category = 'Disassemblers'
+
+VM-Remove-Tool-Shortcut $toolName $category
```

Comment:
After this add in `Uninstall-BinFile -Name $toolName` and it should be gtg after